core/tests/protractor_utils/waitFor.js

```diff
@@ -153,10 +162,12 @@ exports.elementToBeClickable = elementToBeClickable;
 exports.invisibilityOf = invisibilityOf;
 exports.pageToFullyLoad = pageToFullyLoad;
 exports.textToBePresentInElement = textToBePresentInElement;
+exports.presenceOf = presenceOf;
```

Comment:
Why this change? I see a similar code on line 167.

Reply:
That was a duplicate I didn't catch earlier. Removed it